smallrye-reactive-messaging-provider/src/main/java/io/smallrye/reactive/messaging/helpers/Validation.java

```diff
@@ -8,11 +10,11 @@ private Validation() {
 
     public static <T> T notNull(T instance, String name) {
         if (isBlank(name)) {
-            throw new IllegalArgumentException("`name` must be set");
+            throw ex.validationForMustBeSet("name");
```

Comment:
Can we have a "nameMustBeSet" and avoiding passing "name"?

Reply:
I reuse that for other fields (I think), but I can change to have one per field

Reply:
it's just that "name" is often used, so we pass a constant.

Reply:
Changed to dedicated field